src/screens/NewsScreen/Search.js

```diff
@@ -82,7 +82,7 @@ const styles = StyleSheet.create({
     paddingTop: 0,
     paddingBottom: 0,
     alignSelf: 'center',
-    color: COLORS.blackgrey
+    color: COLORS.gray400
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color has been changed from `COLORS.blackgrey` to `COLORS.gray400`. Ensure that `COLORS.gray400` is defined and provides sufficient contrast against the background for readability and accessibility purposes.

```diff
- color: COLORS.blackgrey
+ color: COLORS.gray400
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->